codespell_lib/_codespell.py

```diff
@@ -281,6 +282,12 @@ def parse_options(args):
                       action='store_true', default=False,
                       help='Check file names as well.')
 
+    parser.add_option('--c-escapes',
+                      action='store_true', default=False,
+                      help='Treats files as if they contain C-style character '
+                           'escapes. So for example "\\nHello" is parsed as '
+                           '"hello" instead of "nhallo".')
```

Comment:
nhallo -> nhello

also `\\nHello` -> `\\nhello` or capitalize the other two

And maybe `r"\nhello"` is clearer than `"\\nhello"`?